src/DSP.jl

```diff
@@ -24,6 +21,14 @@ end
 
 @importffts
 
+if VERSION >= v"0.7.0-DEV.602"
+    if isdefined(Base, :DSP)
+        import Base: conv, conv2, deconv, filt, filt!, xcorr
+    else
+        export conv, conv2, deconv, filt, filt!, xcorr
+    end
+end
+
 include("dspbase.jl")
```

Comment:
The latest commit fixes things on SampledSignals and MP3, but it's failing its own tests. Looks like `dspbase.jl` also defines `filt` and `filt!` methods. Should that file even be included for Julia 0.6? AFAICT moving the `include("dspbase.jl")` to inside the `if VERSION >= v"0.7.0-DEV.602"` block above fixes things on 0.6 and 0.7 (nightly)